portfolio/src/main/webapp/script.js

```diff
@@ -38,3 +31,39 @@ async function addQuote() {
 }
 
 
+/**
+ * Adds forum posts to forum
+ */
+async function addForumPosts() {
+
+  const responseFromServer = await fetch('/get-posts');
+  // The json() function returns an object that contains fields that we can
+  // reference to create HTML.
+  const posts = await responseFromServer.json();
+  console.log(posts);
+
+  const bootstrap = 'col-md-4 card me-md-4 mb-md-0 mb-4 p-3 rounded text-center';
+  let textColor = ''
+  let bg = ''
+  
+  // Add it to the page.
+  const forumContainer = document.getElementById('forum-area');
+  
+  for (let i = 0; i < posts.length; i++) {
+    const element = posts[i];
+    let div = document.createElement('div');
+    if (element.secret) {
+      bg = ' bg-warning';
+      textColor = ' text-light';
+    } else {
+      bg = ' bg-light';
+      textColor = ' text-dark';
+    }
+
+    let tags = bootstrap + bg + textColor;
+    div.setAttribute('class', tags);
+    div.innerHTML = "<p>" + element.message + "<br>-" + element.firstName + " " + element.lastName + "</p>";
```

Comment:
[4.4](https://google.github.io/styleguide/jsguide.html#formatting-column-limit): JavaScript code has a column limit of 80 characters.

Apart from breaking up this line you might consider one or more of these JS features:
- [Destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) to unwrap the fields as constants themselves.
- [Template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) to make formatting strings easier
- `document.createElement('p')`

Reply:
Also:

[5.6.1](https://google.github.io/styleguide/jsguide.html#formatting-column-limit): Use single quotes